src/main/resources/elasticsearch/addOnInfoAndVersions-mappings.json

```diff
@@ -14,7 +14,8 @@
         "type": "text"
       },
       "tags": {
-        "type": "keyword"
+        "type": "keyword",
+	"index": "not_analyzed"
```

Comment:
Is this necessary?

https://www.elastic.co/guide/en/elasticsearch/reference/current/keyword.html says:
> Keyword fields are only searchable by their exact value.